### DISCUSSION TITLE
When using CLI mode, is there any method to save the config.yaml file in the checkpoint dir path?

### DISCUSSION THREAD

**Discussion by zengchang233** [Lightning-AI/pytorch-lightning#19988] [General]
I am using pytorch-lightning CLI mode with WandbLogger. I found the config.yaml was always saved in the logger.save_dir. But I would like to save the config file in the checkpoint dir. Does any one have idea about it?

**Comment by mauvilsa** [comment]
You can subclass `SaveConfigCallback` to customize it, in particular see
https://github.com/Lightning-AI/pytorch-lightning/blob/cec6ae123de749927ca42d616e98bb030d54cad7/src/lightning/pytorch/cli.py#L285-L300
